{{cookiecutter.project_slug}}/{{cookiecutter.project_module}}/settings.py

```diff
@@ -26,6 +26,7 @@ class Default:
     DEBUG = True
     SECRET_KEY = os.urandom(24)
     LOGLEVEL = "DEBUG"
+    CORS_ORIGINS = ["http://localhost"]
```

Comment:
How about reading it from environment here too, since it's configurable in .env? If not I would consider removing the entry in docker-compose.yml, but I guess I prefer the former, either keeping `*` or replacing it wiht `localhost`.